pcl_example_lccp_segmentation fails to build

On MacOS 10.9, I get the following errors:
.../examples/segmentation/example_lccp_segmentation.cpp:524:80: error: no member named 'str' in namespace 'boost'
.../examples/segmentation/example_lccp_segmentation.cpp:524:92: error: no member named 'format' in namespace 'boost'

The boost header needs to be added. See https://github.com/nalt/pcl/commit/ddb9b3b659a210761926bd0743f2180a0bbed54b

--- a/examples/segmentation/example_lccp_segmentation.cpp
+++ b/examples/segmentation/example_lccp_segmentation.cpp
@@ -40,6 +40,7 @@
 #include <cmath>
 #include <limits.h>
 
+#include <boost/format.hpp>
 
 
 // PCL input/output

